### DISCUSSION TITLE
posts/jekyll-docs-site/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#212] [Comments]
# posts/jekyll-docs-site/

Documentation for Techno Tim YouTube videos for all your copy pasta needs!

https://technotim.live/posts/jekyll-docs-site/

**Comment by ghost** [comment]
If possible, a tutorial for adding Jekyll plugins for sitemap and SEO would be great!

**Reply by timothystewart6** [reply]
Thanks!  You can always take a look at my site, it's open source! https://github.com/techno-tim/techno-tim.github.io

**Comment by Tom-Whi** [comment]
It’d be awesome to see a video or article how you keep this blog/docs site up to date. I’m running chirpy after seeing your video but I’m struggling to merge the upstream changes into my site without destroying it.  I know there’s plenty of videos documenting the process but having one to follow on from this article would be great!

**Reply by timothystewart6** [reply]
It's outlined on the chirpy site!  You just have to compare your current version to the updated version and then make all the changes :)

**Reply by Tom-Whi** [reply]
Thanks Tim - I did follow their guide but maybe this where my newb understanding of git came in that I was potentially overwriting files that were modified by me (like the config file), and I ended up taking my blog offline for weeks and had to get a friend to roll me back when it all went wrong. I’ll probably just have to leave it at the version it’s at because I’m too worried about breaking it again now I’m too far behind.  Thanks though

**Comment by dwesolowski** [comment]
Working with this theme also, how did you get the left/right footer alignment?

**Comment by raghavduddala** [comment]
Thanks for the video and this blog!!
 I followed both the video and the blog but I run into an issue where the generated site home page just has `--- layout: home # Index page ---` this displayed ( i.e the material from index.html, tried numerous ways of fixing this. When viewed locally, it doesn't have issues. 
Any help is appreciated

**Comment by 1eyeITguy** [comment]
Thanks for this great video and writeup.   I just got it up and running, and I'd love to know how you customized the navigation on left.  Removing the email icon, adding the links link, and adding thing like the shop and sponsor.

Thanks again for all the fantastic content.  I've been able to learn quite a bit from ya.

**Reply by odhyp** [reply]
You can remove the email icon on the sidebar by commenting the `email` option in `_data/contact.yml`. To add a new page like `Sponsor`, make a new markdown file under `_tabs` and use the Front Matter like the other tabs, e.g. `About`, `Categories`, etc.

**Comment by mikemjm** [comment]
This was working great last year, but now I cannot get it working again. GitHub action fails at step "After creating a site based on the template", I can't find a solution.

**Reply by Tom-Whi** [reply]
I ended up sacking off my site and starting again because I was so far behind on the upstream commits.  I found that I no longer needed my "gh-pages" branch and that it seems to have changed something with Github actions.  Once I wiped my repo and re-forked the main from cotes2020 I was able to add my pages and manually merge (offline with Notepad++) my config file, and got back to roughly the same spot I left it in.  Not the most professional way but as an infra/networking monkey it was the best I could do with the limited knowledge I have about git and the template.

**Comment by abdelaziz009** [comment]
hello tim, thanks for the usfull information, how to add login page to the website. "basic authentication"
